archetypes/helidon/src/main/archetype/common/files/src/main/java/module-info.java.mustache

```diff
@@ -0,0 +1,6 @@
+
+module {{package}} {
+{{#module-info}}
+{{.}}
```

Comment:
You should make the model just be the module names.
Also the let's rename `module-info` to just `modules`.


```
module {{package}} {
{{#modules}}
    require {{.}};
{{/modules}}
}
```